packages/framework-provider-kubernetes/src/services/event-registry.ts

```diff
@@ -32,4 +32,19 @@ export class EventRegistry {
     // TODO: Implement accordingly to Dapr
     throw new Error('EventRegistry#queryLatest: Not implemented yet')
   }
+
+  private eventKey(event: EventEnvelope): string {
+    //ee_Post_95ddb544-4a60-439f-a0e4-c57e806f2f6e_event_PostCreated_2021-03-17T16:49:29.143Z_j9qn8kd8
+    const hash = Math.random().toString(36).substring(5)
```

Comment:
This is not 100% random, and it could be possible that in the same ms two similar values could appear. Can you replace with UUID.generate() ?